### ISSUE TITLE
OAuth integration

### ISSUE THREAD

**Issue by SnoopJ** [jay-khatri/Hack-Rice#3]
It'd be nice to have GitHub's [OAuth](https://developer.github.com/guides/basics-of-authentication/) used for generating access tokens instead of requiring intervention on the user's part.

The Chrome extension documentation has [an example](https://developer.chrome.com/extensions/tut_oauth) that looks like it should be helpful in accomplishing this.

**Comment by jay-khatri** [comment]
@SnoopJeDi I'll take a look at this over the weekend.

**Comment by SnoopJ** [comment]
Awesome @jay-khatri! I've let this project stagnate a little, but it could definitely use some polish. Let me know here if you run into any trouble or want somebody to bounce ideas off of.

**Comment by jay-khatri** [comment]
@SnoopJeDi definitely! I saw two things that I'd like to work on. One was OAuth and the other was refactoring some of structure for readability. Thoughts? Also, for publicity sake, Github doesn't provide personalized suggestions do they (I couldn't find anything other than [github.com/explore](url))"?

**Comment by SnoopJ** [comment]
Nope, that's the only available suggestion-y feature I'm aware of, and as near as I can tell it does not seem to be personalized, so GitStub is still providing a relevant feature! 😁
